apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -78,6 +78,7 @@ def create(
         *,
         schema: pa.Schema,
         index_column_names: Sequence[str] = (SOMA_JOINID,),
+        domains: Optional[Sequence[Optional[Tuple[Any, Any]]]] = None,
```

Comment:
given how fundamental this is to the operation of the DataFrame, suggest adding a paragraph detailing the model in the class docstring.